### PR TITLE
Create ManagingQuotasPerProject.md

### PR DESCRIPTION
**Pull request by slinaresGX**
New article added.

### CODE REVIEW

**Review by MaiaShuster — COMMENTED** [review]
Approved.

File: saia-docs/docs/ManagingQuotasPerProject.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* Unit: Cost(USD) or request.
+
+![image](https://github.com/genexus-books/Saia/blob/4f821949d53e8c80233ea10e125cb6745ae14c57/saia-docs/assets/images/Membership_Unit_2.png)
+
+When a Membership is assigned a Freemium value, it means that the membership allows a one-time use, a free trial or a specific
+limit of requests or costs, without an expiration date.
+
+Furthermore, you can define a **Soft limit alert** and a **Hard limit** for each quota limit.  
+
+![image](https://github.com/genexus-books/Saia/blob/4f821949d53e8c80233ea10e125cb6745ae14c57/saia-docs/assets/images/SoftLimit_HardLimit_3.png)
```
Comment (slinaresGX) [inline saia-docs/docs/ManagingQuotasPerProject.md:33]: https://github.com/genexus-books/Saia/blob/4f821949d53e8c80233ea10e125cb6745ae14c57/saia-docs/assets/images/SoftLimit_HardLimit_3.png?raw=true

File: saia-docs/docs/ManagingQuotasPerProject.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+A quota limit can have three statuses: Active, Expired, or Empty. When the set **Hard limit** is reached, the quota status is 
+changed to Empty.
+
+For projects with active quotas, it is not possible to define new quotas. In this situation, it is necessary to edit the 
+active quota and change its status to Expired. 
+
+In addition, you will be able to see the amount of available quota and the amount used:
+
+![image](https://github.com/genexus-books/Saia/blob/4f821949d53e8c80233ea10e125cb6745ae14c57/saia-docs/assets/images/Used_Remaining_4.png)
```
Comment (slinaresGX) [inline saia-docs/docs/ManagingQuotasPerProject.md:53]: https://github.com/genexus-books/Saia/blob/4f821949d53e8c80233ea10e125cb6745ae14c57/saia-docs/assets/images/Used_Remaining_4.png?raw=true

**Review by slinaresGX — COMMENTED** [review]
Los enlaces de las imágenes están mal. Deberían terminar en '?raw=true'.

**Review by MaiaShuster — APPROVED** [review]
Approved.

**Pull request merged**